### PR TITLE
Front to back connection

### PR DESCRIPTION
**Pull request by Atwazar**
This pull request is updating the front with a fonction that is abble to fetch based on the JWT token
it will fectch data, if the JWT token is not on date anymore, it will make the refresh request to get a new one, then  it will try it again.

Tokens are stored in cookies now, as local storage is apparently not safe enough.

This commit is bypassing the 2FA security as the JWT token send by 2FA are not working or error is occuring in the backend.

Finally this commit starts to connect the dots, making a request in the loadpage function, and having some fetch function prepared for the apis

### CODE REVIEW

**Review by BenjaminHThomas — CHANGES_REQUESTED** [review]
We shouldn't be returning 200 "everything's good" when the function fails.

Instagram and other sites return 200 because they're serving a JSON payload that includes the actual status/error message.

**Comment by Atwazar** [comment]
In this PR, the logic for friend is kinda done as well for the profile, 
there's still some redirection issues where the pages is not load automatically but the adress changes, 
to test it you can create multiple accounts via the front end and test to add them, view their profile and delete them, block them, or whatever ( as i said sometimes to go to a profile you have to load the address multiple times)
also when creating an account the error message might be displayed but the account is still created

File: pong-game/game-service/app/user_service/views.py [hunk 1]
```diff
@@ -1,357 +1,427 @@
...
+				"avatar": profile.avatar.url,
+				"isCurrent": user == profile,
+				"isFriend": user.is_friend(profile),
+				"hasBlocked": user.has_blocked(profile),
+				"isSent": user.is_sent(profile),
+				"isPending": user.is_pending(profile)
+			}
+		return Response({
+			"profile": profileData
+		}, status=200)
```
Comment (BenjaminHThomas) [inline pong-game/game-service/app/user_service/views.py:328]: this should be a get request not a post request.

**Comment by Atwazar** [comment]
image
image
changed !

**Review by BenjaminHThomas — APPROVED** [review]
not sure how you managed to convince git that you changed every single line of the views.py, I'm impressed.

LGTM

**Pull request merged**